### PR TITLE
Feature/team page

### PR DESCRIPTION
**Pull request by YrPryhoda**

### CODE REVIEW

File: client/src/containers/TeamPage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from 'react';
+import { Card, Grid, Message, Icon, Header, Image, Button } from 'semantic-ui-react';
+import { Link } from 'react-router-dom';
+import styles from './styles.module.scss';
+import { teamMembers } from 'services/team.service';
+import worksImg from 'assets/images/team-page-works.jpg';
+import linksImg from 'assets/images/team-page-links.jpg';
+
+export const TeamPage: React.FC = () => {
```
Comment (sahanr) [inline client/src/containers/TeamPage/index.tsx:9]: Create separate components for small blocks from this page

File: client/src/containers/Routing/index.tsx [hunk 2]
```diff
@@ -26,6 +27,7 @@ const Routing: React.FC = () => {
 				<PublicRoute exact restricted={true} path="/header" component={Header} />;
 				<PublicRoute exact restricted={true} path="/signup" component={SignUp} />
 				<PrivateRoute exact path="/projects" component={Projects} />
+        <PrivateRoute exact path="/team" component={Team} />
```
Comment (sahanr) [inline client/src/containers/Routing/index.tsx:30]: Use prettier to align the code properly

File: client/src/containers/TeamPage/index.tsx (hunk 1 above)
Comment (YrPryhoda) [reply client/src/containers/TeamPage/index.tsx:10]: fixed

File: client/src/containers/Routing/index.tsx (hunk 2 above)
Comment (YrPryhoda) [reply client/src/containers/Routing/index.tsx]: fixed

**Pull request merged**